assets/css/app.css

```diff
@@ -90,62 +88,134 @@
   cursor: pointer;
 }
 
-.container {
-  @apply h-full
+.wrapper {
+  @apply p-32 h-full;
+  background: rgba(248, 250, 245, 1);
+  padding: 30px 0;
+  width: 100%;
 }
+
 .center_form {
   display: flex;
   justify-content: center;
   align-items: center;
 }
 
-.card {
-  @apply block m-12 p-2 bg-white rounded-lg border border-gray-200 shadow-md hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700;
+.bar-chart {
+  @apply m-4 bg-white rounded-2xl border border-gray-200 shadow-md hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700;
+  max-width: 542px;
+  color: rgba(113, 121, 113, 1);
+  font-size: 16px;
+  width: 100%;
+}
```

Comment:
In bar chart we also need to change the date format as per design

**Design:**
<img width="498" alt="image" src="https://github.com/glific/glific/assets/40158831/612439fe-c615-4857-b04b-45472f4ff657">

**Local:**
<img width="493" alt="image" src="https://github.com/glific/glific/assets/40158831/4f622cbc-7a30-45c6-8cd4-e72b2bd28075">